packages/playwright-test/bundles/babel/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "dependencies": {
     "@babel/code-frame": "^7.18.6",
-    "@babel/core": "^7.20.2",
+    "@babel/core": "^7.20.12",
```

Comment:
Now that you are updating babel/core, you should keep this dependency update consistent across babel packages below as much as possible.

Reply:
Sure, updated all the babel packages to make them consistent.